.travis.yml

```diff
@@ -9,3 +9,6 @@ script:
   - yarn run check
   - yarn run flow-coverage
   - yarn run test-coverage && codecov
+  # If there are any warnings left int the code, this build
+  # will fail because Travis sets environment variable 'CI=true'.
```

Comment:
Do you want to add a note about the source of the warnings? Something like "If the CRA bundled eslint detects any warnings in the code, this build will..."

Reply:
You mean to the README? Or the CONTRIBUTING.md file?

Reply:
I have a broken dev environment right now. New laptop. Would you mind doing something like that now that it's been merged?

Reply:
I meant right there on the line, but it was just a side note :D

Do you still want a note or something in the README?